src/molecules/Header/header.module.css

```diff
@@ -1,7 +1,7 @@
 header {
   align-items: center;
   display: flex;
-  padding: 2em;
+  padding: 20px 28px;
```

Comment:
We need to use `calc()` and `var(--layout-base)` here.

Reply:
Done.